client/src/components/application/Examine/RequestInfoHeader.vue

```diff
@@ -101,7 +101,7 @@
               <h3 v-if="submitCount > 0">SUBMIT COUNT: {{submitCount}}</h3>
 
               <span v-if="prev_nr_required">
-                <span v-if="is_editing" :class="{'form-group-error': $v.previousNr.$error}">
+                <span v-if="is_editing && !is_editing" :class="{'form-group-error': $v.previousNr.$error}">
```

Comment:
problem - this condition will never be true, so Previous NR will never show up editable.

Reply:
Oops! Nice catch, I meant to put a different condition there.